bin/model_generator.dart

```diff
@@ -139,35 +141,43 @@ void writeToFiles(
   }
 }
 
-Future<void> generateJsonGeneratedModels({required bool useFvm}) async {
-  ProcessResult result;
-  if (useFvm) {
-    result = Process.runSync('fvm', [
-      'flutter',
-      'packages',
-      'pub',
-      'run',
-      'build_runner',
-      'build',
-      '--delete-conflicting-outputs',
-    ]);
-  } else {
-    result = Process.runSync('flutter', [
-      'packages',
-      'pub',
-      'run',
-      'build_runner',
-      'build',
-      '--delete-conflicting-outputs',
-    ]);
-  }
-  if (result.exitCode == 0) {
-    print('Successfully generated the jsonSerializable generated files');
-    print('');
-  } else {
+Future<void> _runProcess(String command, List<String> args) async {
```

Comment:
this should not be part of the model_generator.dart file. Move this to a separate file that is not exported